libft/inc/libft.h

```diff
@@ -20,7 +20,7 @@
 # include <stdbool.h>
 # include <limits.h>
 
-# ifdef WITH_OPEN
```

Comment:
libft doesn't pass the `norminette` version `3.3.55`

Reply:
:+1: